SpatialGDK/Source/SpatialGDK/Private/LoadBalancing/GridBasedLBStrategy.cpp

```diff
@@ -2,9 +2,11 @@
 
 #include "LoadBalancing/GridBasedLBStrategy.h"
 
-#include "EngineClasses/SpatialNetDriver.h"
 #include "Utils/SpatialActorUtils.h"
 
+#include "EngineClasses/SpatialNetDriver.h"
```

Comment:
should this include be up above the "Utils/..." one?  I could be wrong, but I thought we did alphabetical Spatial files separate from alphabetical engine files?

Reply:
Yep, I think this is me just seeing the `EngineClasses/` prefix and thinking it was an engine include when really it's GDK